src/lib/functions/synchronous.js

```diff
@@ -1,5 +1,7 @@
 // @ts-check
 const { Buffer } = require('buffer')
+const fs = require('fs')
+const pathModule = require('path')
```

Comment:
[sand] We typically use `path` for the variable that holds the `require` call, or we use object destructuring to get individual methods (e.g. `const { join } = require('path')`). Could we try to do one of those here for consistency?